src/test/rustdoc/type-layout.rs

```diff
@@ -0,0 +1,46 @@
+// compile-flags: --show-type-layout -Z unstable-options
+
+// @has type_layout/struct.Foo.html 'Size: '
+// @has - ' bytes'
+pub struct Foo {
+    pub a: usize,
+    b: Vec<String>,
+}
+
+// @has type_layout/enum.Bar.html 'Size: '
+// @has - ' bytes'
+pub enum Bar<'a> {
+    A(String),
+    B(&'a str, (std::collections::HashMap<String, usize>, Foo)),
+}
+
+// @has type_layout/union.Baz.html 'Size: '
+// @has - ' bytes'
+pub union Baz {
+    a: &'static str,
+    b: usize,
+    c: &'static [u8],
+}
+
+// @has type_layout/struct.X.html 'Size: '
+// @has - ' bytes'
+pub struct X(usize);
+
+// @has type_layout/struct.Y.html 'Size: '
```

Comment:
```suggestion
// @has type_layout/struct.Y.html 'Size: 1 byte'
```

Reply:
I think I tried this and it didn't work because the `Size:` and `1 byte` are part of different HTML elements.

Reply:
Well, you can make them different checks then:
```suggestion
// @has type_layout/struct.Y.html 'Size: '
// @has type_layout/struct.Y.html '1 byte'
```

Reply:
Sure, but that's basically what it does already. It ensures that it has `byte` and not `bytes`.

Reply:
Done in 0dc96ae4ab9.